scripts/data_ingestion/record_retrosheet_downloads.py

```
@@ -0,0 +1,272 @@
+        
+        conn.close()
+        
+    except Exception as e:
+        print(f"✗ Error: {e}")
+        return 1
+    
+    return 0
+
+if __name__ == '__main__':
+    import json
+    exit(main())
```

Ruff (0.15.10)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)